### DISCUSSION TITLE
Why I can't import the Official Class into my project?

### DISCUSSION THREAD

**Discussion by Filicss** [community/community#128352] [Programming Help]
### Body

Ass you can see, I import this ChronoField Class into my project, but my IDE (IntelliJ IDEA) just says it's wrong. What's the problem? How can I fix it? Thank you for your comments.

![01](https://github.com/community/community/assets/172025578/e655368b-8a14-489f-a250-92232d1df34f)
![02](https://github.com/community/community/assets/172025578/9a1c58bc-72c6-4d5e-ac27-6fea447ac3a9)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DaviZCodes** [comment]
I also use IntelliJ IDEA and the import works for me.

Make sure you are using the newest version of Java (it says since 1.8) or IntelliJ IDEA. Also, you can try creating a new project/file since your project build might have been corrupted.

**Reply by ebndev** [reply]
Hi @DaviZCodes, thanks for taking the time to answer!

@Filicss if the response above helped, please feel encouraged to mark it as the answer. If not, let us know if you have further questions or post the solution if you've resolved it on your own. It would be helpful for other users who might experience this in the future 😄

**Comment by Filicss** [comment]
Sorry I didn't reply sooner. The answer didn't help me out of the problem,
but I don't struggle into this problem anymore. Since I can't not provide
more details of my IDE environment, so I don't really hope somebody can
solve the problem by just a few words. But thank you for everyone that had
replied to me and give me any help. Thank you.

ebndev ***@***.***> 于2024年6月20日周四 03:45写道：
